pkg/universe.dagger.io/bash/bash.cue

```diff
@@ -2,13 +2,23 @@
 package bash
 
 import (
+	"list"
+
 	"dagger.io/dagger"
 	"dagger.io/dagger/core"
 
 	"universe.dagger.io/docker"
-	"list"
+	"universe.dagger.io/alpine"
 )
 
+// Like #Run, but with a pre-configured container image.
+#RunSimple: #Run & {
+	_buildDefaultImage: alpine.#Build & {
+		packages: bash: _
+	}
+	input: _buildDefaultImage.output
+}
+
```

Comment:
I understand the improvement but I agree with Gerhard, I would prefer a `#Image` definition that would follow the same pattern as `go` package or other package.

Reply:
What’s the pattern in `go` package that you want to enable?